easy_thumbnails/engine.py

```diff
@@ -57,6 +62,9 @@ def save_image(image, destination=None, filename=None, **options):
                 max(image.size) >= settings.THUMBNAIL_PROGRESSIVE):
             options['progressive'] = True
         try:
+            icc_profile = image.info.get('icc_profile')
```

Comment:
I wonder how expensive this is? Perhaps put it inside `if 'keep_icc_profile' in thumbnail_options:` so that it doesn't run unless needed (in case it is expensive)